api/v1alpha1/onload_types.go

```diff
@@ -73,6 +73,19 @@ type OnloadSpec struct {
 	// ImagePullPolicy is the policy used when pulling images.
 	// More info: https://kubernetes.io/docs/concepts/containers/images#updating-images
 	ImagePullPolicy v1.PullPolicy `json:"imagePullPolicy,omitempty"`
+
+	// +optional
+	// ControlPlane allows fine-tuning of the Onload control plane server.
+	ControlPlane ControlPlaneSpec `json:"controlPlane"`
```

Comment:
```suggestion
	ControlPlane *ControlPlaneSpec `json:"controlPlane,omitempty"`
```
Not sure if the pointer is 100% necessary, but `omitempty` is good to have for optional fields.

Reply:
A pointer and `omitempty` would also align with the rest of the file, indeed.